lightning/src/ln/functional_tests.rs

```diff
@@ -4667,27 +4667,15 @@ pub fn bolt2_open_channel_sending_node_checks_part2() {
 
 	let node_b_id = nodes[1].node.get_our_node_id();
 
-	// BOLT #2 spec: Sending node must set funding_satoshis to less than 2^24 satoshis
```

Comment:
Why is this test useless? BOLT 02 does require the receiver to reject channels that go over this limit so seems reasonable enough to keep?